### PR TITLE
Web scraper tothl

### PR DESCRIPTION
**Pull request by tothlilla**

### CODE REVIEW

File: docs/web_scraper.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+We have been creating a program that is able to detect and modify shapes and text on images. This repository is a container of all the Python scripts that have been used for the above mentioned OpenCV project by MEGALOTIS-SAPPHIRE project team.<br/>
+
+List of available scripts, their definitions, and short descriptions:
+
+
+- image download with web scraper
+
+ `downloadimages(query)`<br/>
+
+The 'web_scraper.py' command can be executed in command line. It takes four obligatory arguments as input to implement them into downloadimages(query) function that are able to download images around a given topic and users are able to download as many images as they want. To implement this function, users have to give the following arguments: -k <keyword, as the topic> -n <number of downloading images>. If users need more help, they can use -h/--help argument after 'web_scraper.py' command. The extensions of the downloaded files are .jpg. The command generate a folder called 'download' and each query will be saved in different subfolders with the names of the pre-given keywords.<br/>
```
Comment (znemeth) [inline docs/web_scraper.md]: *is* able to download images *with a certain* topic
and *the* users
if *the* users
*extension* ... *file is*
*generates*

File: web_scraper.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    arguments = {"keywords": query, 
+                 "format": "jpg", 
+                 "limit": x, 
+                 "print_urls":False, 
+                 "size": "medium", 
+                 "aspect_ratio": "panoramic"}
+    try: 
+        path = response.download(arguments) #this is a tuple
+
+    # Handling File NotFound Error
```
Comment (znemeth) [inline web_scraper.py]: remove comment!

File: web_scraper.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                     "format": "jpg", 
+                     "limit": x, 
+                     "print_urls":False,  
+                     "size": "medium"} 
+        try: 
+            response.download(arguments)  
+        except: 
+            pass
+  
+# Driver Code
```
Comment (znemeth) [inline web_scraper.py]: remove comment!

File: web_scraper.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                     "size": "medium"} 
+        try: 
+            response.download(arguments)  
+        except: 
+            pass
+  
+# Driver Code 
+for query in search_queries: 
+    downloadimages(query)
+
```
Comment (znemeth) [inline web_scraper.py:36]: add new line!

**Review by znemeth — APPROVED** [review]

**Review by leventekobor — APPROVED** [review]
looks super fine

**Pull request merged**